File: plugin-fixtures/src/main/java/com/github/cs124_illinois/questioner/testing/addone/incorrect/extraoutput/Question.java
package com.github.cs124_illinois.questioner.testing.addone.incorrect.extraoutput;

import edu.illinois.cs.cs125.questioner.lib.Incorrect;

@Incorrect("extraoutput")
public class Question {
  int addOne(int value) {
    System.out.println(value);
    return value + 1;
  }
}

File: plugin-fixtures/src/main/java/com/github/cs124_illinois/questioner/testing/withfeaturecheck/correct/kotlin/Question.kt
package com.github.cs124_illinois.questioner.testing.withfeaturecheck.correct.kotlin

import edu.illinois.cs.cs125.questioner.lib.AlsoCorrect

/*
 * For testing features.
 */

@AlsoCorrect
class Question {
  fun sumTo(range: Int): Int = range * (range + 1) / 2
}